lib/cylc/network/server.py

```diff
@@ -84,15 +85,15 @@ def __init__(self, encode_method, decode_method, secret_method):
         self.secret = secret_method
 
     def start(self, min_port, max_port):
-        """Start the server running
+        """Start the server running.
 
-        Args:
-            ports (iterable): Generator of ports (int) to choose from.
-                The lowest available port will be chosen.
+        Will use a port range provided to select random ports.
 
+        Args:
+            min_port (int): minimum socket port number
+            max_port (int): maximum socket port number
```

Comment:
BTW: we should make the site configuration more compatible with this, doing `min(ports), max(ports)` is wrong.

Reply:
Puted to #2978